api/api.ts

```diff
@@ -1,12 +1,10 @@
 import axios from "axios";
 import { Buffer } from "buffer";
-import { URL, CERTIFICATE_URL } from "@env";
-
 const timeoutInMs = 1200;
 
 // move these to .env file next sprint
-const url = URL;
-export const certificateUrl = CERTIFICATE_URL;
+const url = process.env.URL;
```

Comment:
@Danar435 Do this globally. Replace all imports from `@env` with `process.env....`